README.md

```diff
@@ -2,11 +2,46 @@
 
 This repository is a command line tool for testing Azure Policies.
 
-This repository is organized as follow:
+## Usage
 
-- custom_policies : contains 1 folder (1 [terraform Module](https://www.terraform.io/docs/configuration/modules.html)) per custom Azure Policy. 
-- prod_policies : this folder contains the terraform configuration applied on Azure environments to match security requirements.
-- test : for each custom Azure Policy, automated tests are located here.
+For each test case, the following directory structure must be created:
+
+```txt
++-- your_test_name
+|   +-- setup
+|   |   +-- terraform files to setup the policy
+|   +-- terraform files for test cases
+```
+
+Additionally, a `.yaml` configuration file must describe the test as following:
+
+```yaml
+name: Name of the test
+cases:
+- variables:
+  - key: variable name in TF
+    value: val
+  wantErr: true
+- variables:
+  - key: variable name in TF
+    value: val2
+  wantErr: false
```

Comment:
instead of wantErr I'd propose errorExpected, what do you think ?

Reply:
done